types/relay-compiler/lib/RelayCompilerPublic.d.ts

```diff
@@ -0,0 +1,3 @@
+import * as GraphQLCompilerTypes from "graphql-compiler";
```

Comment:
graphql-compiler was merged in relay-compiler

should we remove graphql-compiler at all?

Reply:
we have not a dependency `graphql-compiler` ... it is transitive from `relay-compiler`. But we need the typings `graphql-compiler`, then I just re-exported them as `RelayCompilerPublic`.